deploy/rbd/helm/templates/nodeplugin-clusterrole.yaml

```diff
@@ -22,4 +22,7 @@ rules:
   - apiGroups: ["storage.k8s.io"]
     resources: ["volumeattachments"]
     verbs: ["get", "list", "watch", "update"]
+  - apiGroups: [""]
+    resources: ["configmaps"]
+    verbs: ["get", "list"]
```

Comment:
I copied these from the static manifests. I don't think we really need these though? They are only used for provisioning, not node driver stuff?

Reply:
configmap is loaded during startup here https://github.com/ceph/ceph-csi/blob/master/pkg/rbd/rbd.go#L111

Reply:
hmm.... is this for its state configmaps?

If so, this should probably be a Role in the deployed namespace so it can be restricted to just the namespace's configmaps rather then all configmaps in the entire cluster.